talk/morelanguage/lambda.tex

```diff
@@ -341,3 +342,4 @@
   \end{exampleblock}
 
 \end{frame}
+\end{advanced}
```

Comment:
This change should be dropped after rebasing on master. This has already been done in another MR (and more slides have been moved to advanced actually)

Reply:
Right, I saw the other PR.